app/assets/stylesheets/geoblacklight/modules/metadata.scss

```diff
@@ -1,5 +1,28 @@
-.metadata-view {
-  pre {
-    @extend .pre-scrollable;
+/**
+ * Rules for the geospatial metadata Bootstrap Modal dialog
+ */
+.modal-dialog {
+  width: 1148px;
+}
+
+.modal-header {
+  .modal-title {
+    font-weight: 400;
+    font-size: 20px;
+  }
+}
+
+.modal-body {
+  overflow-y: scroll;
```

Comment:
What do you think about constraining the body height so that that the entire modal fits on the screen? It would be nice to see that download is an option without scrolling to the bottom of the page. Here's what it looks like when you style  `modal-body` with `height: calc(100vh - 190px)`:
<img width="1430" alt="screenshot 2017-08-09 00 06 44" src="https://user-images.githubusercontent.com/784196/29105920-3fa36cfe-7c97-11e7-9744-55d8acc6462f.png">